### ISSUE TITLE
Monkey Patching breaks defining relations models in app with custom user model

### ISSUE THREAD

**Issue by emperorcezar** [django-guardian/django-guardian#219]
In [models.py](https://github.com/lukaszb/django-guardian/blob/v1.2/guardian/models.py#L92) monkey patching calls get_user_model.

According to the [docs](http://django-guardian.readthedocs.org/en/v1.2/userguide/performance.html) on performance you can define a model that relates the object to the user.

If you create this model in the same app as the custom user model it will cause the following error: 
 
```
django.core.exceptions.ImproperlyConfigured: AUTH_USER_MODEL refers to model 'myapp.MyCustomUser' that has not been installed
```

I believe a good solution to this is to add a setting to disable monkey patching since we've already using a custom user model, we can use a mixin to provide the needed functionality.

If that sounds good I'll write it up and open a pull request?

**Comment by brianmay** [comment]
Can this issue be closed? #220, now merged, supports disabling of monkey patching.

**Comment by emperorcezar** [comment]
Yes, this can be closed.
